### PR TITLE
Chore: upgrade to 2.6.16

### PR DESCRIPTION
**Pull request by till**
Probably related: ~~robertdebock/molecule-action#5~~

---

[docker-5.0.0](https://pypi.org/project/docker/) removed Python 2.7.

### CODE REVIEW

**Comment by rwos** [comment]
Okay, so 3.1.4 has the same error. The import error seems to come from `websockets`, via `docker`. ~~So probably docker is pinned, but it installs a newer `websockets` pip package or so...~~ `websockets` has always been python 3+, so can't be that. Still looking

**Comment by rwos** [comment]
https://github.com/docker/docker-py/issues/2840#issuecomment-843814192

**Comment by rwos** [comment]
k, upgrade fails, locally too

**Comment by till** [comment]
Hmm, I guess I broke it. 😒

**Comment by rwos** [comment]
don't really see any large changes... bit unclear how this worked in https://github.com/hostwithquantum/ansible-docker-plugin-loki/runs/2003025612 but not here

**Review by rwos — APPROVED** [review]

File: library/loki_setup.py [hunk 1]
```diff
@@ -9,18 +9,50 @@
 
 # code dependencies
 import docker
+from docker.models.plugins import Plugin
+from docker.errors import NotFound, APIError, DockerException
+
+
+# This is a work-around because I was trying to smart and use the Docker API.
+def _disable(the_plugin, client):
```
Comment (till) [inline library/loki_setup.py:17]: Maybe one day: https://github.com/docker/docker-py/pull/2843

**Pull request merged**